doc/classes/Slider.xml

```diff
@@ -37,13 +40,30 @@
 			</description>
 		</signal>
 	</signals>
+	<constants>
+		<constant name="POSITION_BOTTOM" value="0" enum="TickPosition">
```

Comment:
Maybe BOTTOM_RIGHT and TOP_LEFT would be better? The current values makes sense only for horizontal sliders.